spec/models/miq_server/log_management_spec.rb

```diff
@@ -149,6 +149,68 @@ def stub_vmdb_util_methods_for_collection_log
     describe "#post_historical_logs" do
       context "Server" do
         include_examples "post_[type_of_log]_logs", "MiqServer", :historical
+
+        context "new tests" do
```

Comment:
This will be cleaned up and put in a nicer place in the automate dialog/models export PR